content/en/docs/components/hyperparameter-tuning/hyperparameter.md

```diff
@@ -121,6 +121,18 @@ example:
     Namespace: kubeflow
     ```
 
+1. (Optional) **Note that** Katib's experiments don't work with
+   [Istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
+   If you installed Kubeflow using [Istio config](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/),
+   you have to disable sidecar injection. To do that, specify annotation `sidecar.istio.io/inject: "false"`
+   in your experiment's trial template.
+
+   For the provided random example with Kubernetes [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+   trial template, annotation should be under
+   [`.trialSpec.spec.template.metadata.annotations`](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/random-example.yaml#L52).
+   For the Kubeflow `TFJob` or other training operators check
+   [here](https://www.kubeflow.org/docs/components/training/tftraining/#what-is-tfjob) how to set annotation.
+
 1. Deploy the example:
     ```
```

Comment:
Thank you @andreyvelich One more thing - again, if you have time - can you add blank lines that separate code blocks and text paragraphs here and elsewhere (e.g. lines 327, 337)
```suggestion

    ```shell
```

Appreciate it.

/assign @gaocegege @johnugeorge @sperlingxx

(OWNERS/approvers other than @andreyvelich 👍)